theano/tensor/tests/test_opt.py

```diff
@@ -6101,7 +6106,8 @@ def test_local_reshape_lift():
     topo = f.maker.fgraph.toposort()
     assert isinstance(topo[-2].op, tensor.Reshape)
     assert isinstance(topo[-1].op, tensor.Elemwise)
-
+    # Check stacktrace was copied over correctly after opt was applied
+    assert check_stack_trace(f, ops_to_check='last')
```

Comment:
I think you should check both the last one (Elemwise) and the previous one (Reshape), that was affected by the optimization as well.